src/components/PlioQuestion.vue

```diff
@@ -274,7 +318,8 @@ export default {
     clickClose() {
       this.closeModal();
       this.isAnswerSubmitted = false;
-      this.$emit("question-closed", this.plioQuestion, this.selectedOption);
+      this.$emit("question-closed", this.plioQuestion, 
+        this.selectedOption, this.updatedProgressBarInfo);
```

Comment:
When `close` button is clicked, along with all other info, we're also passing the info that the progress bar has moved to a new % value so it could be used again in next question

Reply:
Oh no... Just realized. The progress bar calculations are being done inside `PlioQuestion`. This IMO, is wrong, because it assumes that questions know about other questions (`totalQuestions` is the biggest sinner).

The correct design here is to have the progress progress be set from the player:

1. The question doesn't know about other questions
2. The player tells the progress bar how much to do, and the modal just shows it. It's actually okay for the progress bar to know this (though that means it will not be reusable in other places)

If you want, you can skip this today for merging in case it's a lot of work. But this is VERY IMPORTANT (my opinion, of course), for separation of responsibility. You can merge, test, and then make this change immediately as your next task after the A/B testing release.